### PR TITLE
Add concurrency control for tableDb,groupDb and meterDb in OFSwitch

### PR DESCRIPTION
**Pull request by ceclinux**

### CODE REVIEW

**Comment by ceclinux** [comment]
No error is reported when running `make golangci` on my machine.

File: ofctrl/ofSwitch.go [hunk 1]
```diff
@@ -46,15 +46,18 @@ type OFSwitch struct {
 	app    AppInterface
 	// Following are fgraph state for the switch
 	tableDb        map[uint8]*Table
+	tableDbMux     sync.Mutex
```
Comment (wenyingd) [inline ofctrl/ofSwitch.go:49]: Shall we use `RWMutex`?

**Review by tnqn — APPROVED** [review]
Shouldn't it update version file?

**Comment by wenyingd** [comment]
> Shouldn't it update version file?

It does require.

**Review by tnqn — DISMISSED** [review]
LGTM

**Comment by tnqn** [comment]
I don't know why but there are many errors in golangci.

**Comment by ceclinux** [comment]
I think the errors come from the CI itself. Not from this pr https://github.com/antrea-io/ofnet/pull/53

**Comment by wenyingd** [comment]
It can be resolved by bump up golangci version to v1.52.1, I created another pr #54 for it. We can merge this one first, what's your thought @tnqn ?

**Review by wenyingd — APPROVED** [review]
LGTM

**Comment by wenyingd** [comment]
The golangci check has passed after rebase on main.

**Review by tnqn — APPROVED** [review]

**Pull request merged**